### PR TITLE
Added GitHub issue numbers next to relevant TODOs

### PR DESCRIPTION
**Pull request by donwung**
### Short description

Appended GitHub issue numbers next to JIRA tickets. No functional changes.

---

### CODE REVIEW

File: native/src/utils/PushNotificationsManager.ts [hunk 1]
```diff
@@ -102,7 +102,7 @@ export const useForegroundPushNotificationListener = ({
         if (mounted) {
           // The CMS needs some time until the push notification is available in the API response
           setTimeout(() => {
-            // TODO IGAPP-1024: Uncomment and improve snackbar
+            // TODO IGAPP-1024 (#1979): Uncomment and improve snackbar
```
Comment (steffenkleinle) [inline native/src/utils/PushNotificationsManager.ts:105]: Sorry, but it seems like this was a typo. The correct issue numbers would be IGAPP-1204 and #2140

**Review by steffenkleinle — COMMENTED** [review]
Hey @donwung, thanks a lot for your contribution, much appreciated!

**Review by LeandraH — COMMENTED** [review]
Hey, thank you so much for your PR!

There is another TODO that can be replaced, the one to IGAPP-1078. That is an incorrect link to a design ticket, it should actually be to IGAPP-1177, aka #2117.

And there is a TODO to the closed ticket IGAPP-281, I'll open a new issue for us to fix that separately.

**Comment by LeandraH** [comment]
Sorry for the duplicate comments, didn't refresh the page 😅

**Comment by donwung** [comment]
I updated TODOs to replace JIRA codes with GH issue numbers, as suggested by @LeandraH 
And left the TODOs in `createEventsEndpoints.ts` in `api-client\src\endpoints` alone since I suspect @steffenkleinle is working on it

File: web/src/components/KebabMenu.tsx [hunk 2]
```diff
@@ -80,8 +80,6 @@ const Content = styled.div`
 const ToggleButton = styled.button`
   background-color: transparent;
   border: none;
-  /* TODO IGAPP-889 remove this margin after implementing new search icon to fix wrong viewBox of svg */
-  margin-top: 6px;
```
Comment (steffenkleinle) [inline web/src/components/KebabMenu.tsx:84]: I am sorry for the confusion but actually this is still needed for now :/ However, I am already working on it in #1882, so could you please readd the margin and just change the issue number to #1882? Thanks!

**Review by SteffiStuffel — APPROVED** [review]
Looks good. Thanks for your contribution :tada:

**Review by steffenkleinle — APPROVED** [review]

**Comment by SteffiStuffel** [comment]
Thanks for this PR, i moved your commits here: https://github.com/digitalfabrik/integreat-app/pull/2325

For security reasons, it is forbidden in our repro to start our CI checks from a fork, therefore I moved them.

**Pull request closed without merging**